pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -106,6 +106,7 @@ type DomainManager interface {
 	GetUsers() ([]v1.VirtualMachineInstanceGuestOSUser, error)
 	GetFilesystems() ([]v1.VirtualMachineInstanceFileSystem, error)
 	SetGuestTime(*v1.VirtualMachineInstance) error
```

Comment:
Can `SetGuestTime` be moved out of the `DomainManager` and changed to be private?

Reply:
Yes I think we can do that

Reply:
Done